src/db_queries/expenses.ts

```diff
@@ -8,23 +8,23 @@ const helpers = new Helpers();
 
 export class Expenses {
 
-    _readExpenseQuery = "SELECT * FROM organizations_centers_expenses WHERE center_id = $1";
-    _createExpenseQuery = "INSERT INTO organizations_centers_expenses (center_id, month, year, expense, amount, is_recurring) VALUES ($1, $2, $3, $4, $5, $6)";
-    _deleteExpenseQuery = "DELETE FROM organizations_centers_expenses WHERE center_id = $1 AND id = $2";
-    _updateExpenseQuery = "UPDATE organizations_centers_expenses SET month = $1, year = $2, expense = $3, amount = $4, is_recurring = $5 WHERE center_id = $6 AND id = $7";
-    _totalExpenseQuery = "SELECT SUM(amount) FROM organizations_centers_expenses WHERE center_id = $1";
-    _totalExpensePaidQuery = "SELECT SUM(amount) FROM organizations_centers_expenses_paid WHERE center_id = $1";
+    private readExpenseQuery = "SELECT * FROM organizations_centers_expenses WHERE center_id = $1";
+    private createExpenseQuery = "INSERT INTO organizations_centers_expenses (center_id, month, year, expense, amount, is_recurring) VALUES ($1, $2, $3, $4, $5, $6)";
+    private deleteExpenseQuery = "DELETE FROM organizations_centers_expenses WHERE center_id = $1 AND id = $2";
+    private updateExpenseQuery = "UPDATE organizations_centers_expenses SET month = $1, year = $2, expense = $3, amount = $4, is_recurring = $5 WHERE center_id = $6 AND id = $7";
+    private totalExpenseQuery = "SELECT SUM(amount) FROM organizations_centers_expenses WHERE center_id = $1";
```

Comment:
Plural works better here: `_totalExpensesQuery` instead of `_totalExpenseQuery` and `_totalExpensesPaidQuery` instead of `_totalExpensePaidQuery`.